Adding new intents + training examples (#56)
RasaHQ/NLU-training-data: banking/nlu.md

 - Can I see my inflows and outflows?
 - Please show me my transaction history
 - Please retieve my transaction log
 - I would like to see my bank account statement history

# intent:interest_rate <!--The user would like to get some information on interest rates.-->

 - yeah hi what is the current interest rate for loans
 - what's the interest rate for a 30 year mortgage
 - what's the availavble interest rate for mortgages today
 - what's the interest rate for your golden credit cards 
 - what's the interest rate for your credit card
 - What's the best interest rate I can get within your bank
 - I need to borrow some money so I want to know the interest rates
 - What are the interest rates if I ask for one hundred thousand

> # intent:lost_card <!--The user lost his/her card-->

Reviewer: ```suggestion
# intent:lost_card <!--The user lost their card-->
```

 - i've lost my card 
 - I don't know where my card is
 - I have no idea where've left my card
 - I left my card somewhere and now I need to cancel it
 - I really don't remember where my card is
 - I'm afraid I don't know where my card is
 - I've lost my credit card and I don't think I will find it again
 - My card is gone I must have left it somewhere
 - I'm afraid someone took my card so I'd better cancel it
 - I have no clue where is my card 
 - Can't find my card don't know what to do
 - Can't find my card what should I do

> # intent:change_pin <!--The user wants to change his/her pin code-->

Reviewer: ```suggestion
# intent:change_pin <!--The user wants to change their pin code-->
```
